### PR TITLE
feat: authentication(login)

### PR DESCRIPTION
**Pull request by PeeyushPrashant**
- login page
- used encodedToken
- implemented using localStorage

Screenshot (67)

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -3,15 +3,19 @@ import logo from "./logo.png";
 import Mockman from "mockman-js";
 import LandingPage from "./pages/LandingPage/Landingpage";
```
Comment (anuj-kumary) [inline src/App.js]: For Pages You can use barrel import.

File: src/App.js [hunk 2]
```diff
@@ -3,15 +3,19 @@ import logo from "./logo.png";
...
 function App() {
   return (
     <div className="App">
       <Routes>
         <Route path="/" element={<LandingPage />} />
         <Route path="/products" element={<ProductList />} />
+        <Route path="/cart" element={<Cart />} />
+        <Route path="/login" element={<Login />} />
       </Routes>
+      {/* <Login /> */}
```
Comment (anuj-kumary) [inline src/App.js]: Remove unwanted comments

File: src/pages/CartPage/Cart.jsx [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import "./Cart.css";
+
+const Cart=()=>{
```
Comment (anuj-kumary) [inline src/pages/CartPage/Cart.jsx]: Why this components empty?

File: src/pages/Login/Login.jsx [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+import axios from "axios"
+import {  useNavigate } from "react-router-dom"
+const Login = ()=>{
+  const {setToken}=useToken();
+  const testCredentials= { email: "adarshbalika@gmail.com",password: "adarshbalika"};
+  let navigate= useNavigate();
+  const loginHandler=async()=>{
+    try{
+      const response = await axios.post('/api/auth/login', {...testCredentials});
+      if(response.status===200)
```
Comment (anuj-kumary) [inline src/pages/Login/Login.jsx]: I think when we are doing post method it's status code 201. Or you can give both here 200 and 201

Comment (PeeyushPrashant) [reply src/pages/Login/Login.jsx]: ya actually I was getting status as 200, I'll provide both

**Review by anuj-kumary — APPROVED** [review]
Your code is very clean also you used good naming conventions.
Their is some suggestion just look into it.

File: src/pages/Login/Login.jsx (hunk 4 above)
Comment (MurtazaBagwala07) [inline src/pages/Login/Login.jsx]: for post , status code generally is 201

**Review by MurtazaBagwala07 — APPROVED** [review]
Code is clean and readable
Folder structure looks modular

**Pull request merged**